### PR TITLE
AT_01.02_004 | User is able to get Search Box by a keyboard shortcut (Ctrl+K)

### PR DESCRIPTION
**Pull request by DimaCurkanu**
https://trello.com/c/CFpFDHEm/1976-rf-pom-at0102004-user-is-able-to-get-search-box-by-a-keyboard-shortcut-ctrlk

### CODE REVIEW

**Comment by DimaCurkanu** [comment]
[[RF] POM AT_01.02_004 | User is able to get Search Box by a keyboard shortcut (Ctrl+K)](https://trello.com/c/CFpFDHEm/1976-rf-pom-at0102004-user-is-able-to-get-search-box-by-a-keyboard-shortcut-ctrlk)

File: cypress/pageObjects/SearchResultPage.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class SearchResultPage {
```
Comment (Rom-T) [inline cypress/pageObjects/SearchResultPage.js:1]: У нас уже есть такой класс ResultSearchBoxPage.js

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 2]
```diff
@@ -238,4 +240,15 @@ describe('headerAndFooter', () => {
                 .getFreestyleProjectHeader()
                 .should('contain', `Project ${searchBoxDocumentationPageData.setNamesForNewProject[0].name}`)
       })
+
+      it('AT_01.02_004 | User is able to get Search Box by a keyboard shortcut (Ctrl+K)', function () {
+        homePage
+        .openSearchByShortCut()
+        .typeSearchBoxInputField(searchBoxDocumentationPageData.multibranchPipeline.name + '{enter}')
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:247]: Please add 4 spaces/1 tab for methods

File: cypress/pageObjects/HomePage.js [hunk 3]
```diff
@@ -466,5 +466,11 @@ class HomePage {
     return this
     }
 
+    openSearchByShortCut() {
+        this.getPageBody()
+        .trigger('keydown', { ctrlKey: true, keyCode: 75 });
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:471]: no need to wrap to new line

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 2 above)
Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js:247]: Done

File: cypress/pageObjects/SearchResultPage.js (hunk 1 above)
Comment (DimaCurkanu) [reply cypress/pageObjects/SearchResultPage.js:1]: References changed. File SearchResultPage deleted

File: cypress/pageObjects/HomePage.js (hunk 3 above)
Comment (DimaCurkanu) [reply cypress/pageObjects/HomePage.js:471]: Done. Thank you!

**Review by Rom-T — APPROVED** [review]

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**